gaffer-as-a-service/pom.xml

```diff
@@ -41,6 +41,7 @@
         <fasterxml.jackson.version>2.11.2</fasterxml.jackson.version>
         <javax.xml.bind.version>2.3.0</javax.xml.bind.version>
         <slf4j.api.version>1.7.32</slf4j.api.version>
+        <log4j.version>1.2.17</log4j.version>
```

Comment:
You did make this log4j2 and added a log4j2.properties but now the version is back to 1.2.17

Reply:
Please review @t92549 
As I updated the pom.xml with the log4j2 stable version and dependencies. Tested/debuged the code and the logging works fine.